doc/api/fs.md

```diff
@@ -4652,1174 +3925,1920 @@ On Linux, positional writes don't work when the file is opened in append mode.
+`Date` and number values are not connected. Assigning a new number value, or
+mutating the `Date` value, will not be reflected in the corresponding alternate
+representation.
+
+The times in the stat object have the following semantics:
+
+* `atime` "Access Time": Time when file data last accessed. Changed
+  by the mknod(2), utimes(2), and read(2) system calls.
+* `mtime` "Modified Time": Time when file data last modified.
+  Changed by the mknod(2), utimes(2), and write(2) system calls.
+* `ctime` "Change Time": Time when file status was last changed
+  (inode data modification). Changed by the chmod(2), chown(2),
+  link(2), mknod(2), rename(2), unlink(2), utimes(2),
+  read(2), and write(2) system calls.
+* `birthtime` "Birth Time": Time of file creation. Set once when the
+  file is created. On filesystems where birthtime is not available,
+  this field may instead hold either the `ctime` or
+  `1970-01-01T00:00Z` (ie, Unix epoch timestamp `0`). This value may be greater
+  than `atime` or `mtime` in this case. On Darwin and other FreeBSD variants,
+  also set if the `atime` is explicitly set to an earlier value than the current
+  `birthtime` using the utimes(2) system call.
+
+Prior to Node.js 0.12, the `ctime` held the `birthtime` on Windows systems. As
+of 0.12, `ctime` is not "creation time", and on Unix systems, it never was.
+
+### Class: `fs.WriteStream`
 <!-- YAML
-added: v10.0.0
+added: v0.1.93
 -->
 
-* `path` {string|Buffer|URL}
-* `atime` {number|string|Date}
-* `mtime` {number|string|Date}
-* Returns: {Promise}
+* Extends {stream.Writable}
 
-Change the file system timestamps of the object referenced by `path` then
-fulfills the `Promise` with no arguments upon success.
+Instances of {fs.WriteStream} are created and returned using the
+[`fs.createWriteStream()`][] function.
 
-The `atime` and `mtime` arguments follow these rules:
+#### Event: `'close'`
+<!-- YAML
+added: v0.1.93
+-->
 
-* Values can be either numbers representing Unix epoch time, `Date`s, or a
-  numeric string like `'123456789.0'`.
-* If the value can not be converted to a number, or is `NaN`, `Infinity` or
-  `-Infinity`, an `Error` will be thrown.
+Emitted when the {fs.WriteStream}'s underlying file descriptor has been closed.
 
-### `fsPromises.writeFile(file, data[, options])`
+#### Event: `'open'`
 <!-- YAML
-added: v10.0.0
-changes:
-  - version: v14.17.0
-    pr-url: https://github.com/nodejs/node/pull/35993
-    description: The options argument may include an AbortSignal to abort an
-                 ongoing writeFile request.
-  - version: v14.12.0
-    pr-url: https://github.com/nodejs/node/pull/34993
-    description: The `data` parameter will stringify an object with an
-                 explicit `toString` function.
-  - version: v14.0.0
-    pr-url: https://github.com/nodejs/node/pull/31030
-    description: The `data` parameter won't coerce unsupported input to
-                 strings anymore.
+added: v0.1.93
```

Comment:
```suggestion
added:
  - v10.0.0
  - v0.1.93
changes:
  - version: v14.17.0
    pr-url: https://github.com/nodejs/node/pull/35993
    description: The options argument may include an AbortSignal to abort an
                 ongoing writeFile request.
  - version: v14.12.0
    pr-url: https://github.com/nodejs/node/pull/34993
    description: The `data` parameter will stringify an object with an
                 explicit `toString` function.
  - version: v14.0.0
    pr-url: https://github.com/nodejs/node/pull/31030
    description: The `data` parameter won't coerce unsupported input to
                 strings anymore.
```

https://github.com/nodejs/node/blob/52abf271c563ddffdc93b444ea05e5347a7f2784/doc/api/fs.md#L6131-L6146

Reply:
Hum this change is incorrect, I'll be sending a PR to fix the `master` version, please disregard.